### PR TITLE
[fix](tvf)support s3,local compress_type and append regression test

### PR DESCRIPTION
**Pull request by hubgeter**
## Proposed changes

Issue Number: close #xxx
support s3,local compress_type and append regression test.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

**Review by Hastyshell — APPROVED** [review]
LGTM

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/suites/external_table_p2/tvf/test_s3_tvf_compression.groovy [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    String bucket = context.config.otherConfigs.get("s3BucketName");
+    
+    sql """ set query_timeout=3600; """ 
+
+    String compress_type = "gz" 
+    qt_gz_1 """ 
+    select * from 
+    s3(     
+        "URI" = "https://${bucket}.${s3_endpoint}/regression/tvf/compression/test_tvf.csv.${compress_type}",    
+        "ACCESS_KEY" = "${ak}",
```
Comment (morningman) [inline regression-test/suites/external_table_p2/tvf/test_s3_tvf_compression.groovy:33]: Change some of cases to the new type of properties:
```
"s3.access_key"= "minioadmin",
"s3.secret_key" = "minioadmin",
"format" = "csv",
"use_path_style" = "true"
```

**Comment by hubgeter** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**